build.sh

```diff
@@ -244,7 +244,12 @@ while [[ $# -gt 0 ]]; do
         ;;
 
     --with-intl)
-        INTL_ICU="-DINTL_ICU_SH=1"
+        # todo: remove me! this is temporary
+        # until new setting settles and we re-configure CI
```

Comment:
Re-configuring CI is as simple as removing the --with-intl flag that I added to netci.groovy

Reply:
Indeed but reconfiguration applies to next PR not this one.. so.. in order this one to pass the tests, I need this change.